tests/e2escenarios/e2e_images_test.go

```diff
@@ -113,4 +113,23 @@ var _ = Describe("odo supported images e2e tests", func() {
 			verifySupportedImage(filepath.Join("rhscl", "nodejs-10-rhel7:latest"), "nodejs", "nodejs:latest", project, appName, context)
 		})
 	})
+
+	Context("odo supported private registry images deployment", func() {
+		JustBeforeEach(func() {
+			if os.Getenv("CI") != "openshift" {
+				Skip("This is an openShift CI specific scenario, skipping")
```

Comment:
@prietyc123 Registry secrets are by default configured in multistage test infra so it works but this test scenario is not specific to 4.* version. It should work on 3.11 cluster too i mean in travis CI it should run too. 

We have not configured custom secret in travis to access private registry. So please create an issue to track it pass a skip message that makes proper sense. For example
```
       // Issue for configuring login secret for travis CI
       if os.Getenv("CI") != "openshift" {
       Skip("Skipping it on travis CI, skipping")
}

Reply:
Tracked the issue https://github.com/openshift/odo/issues/3640 for travis CI.